adapters/handlers/rest/middlewares.go

```diff
@@ -72,6 +72,8 @@ func addClusterHandlerMiddleware(next http.Handler, appState *state.State) http.
 	raftRouter := raft.ClusterRouter(appState.SchemaManager.Handler)
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		switch {
+		case r.URL.Path == "/v1/cluster/statistics":
+			next.ServeHTTP(w, r)
```

Comment:
**question (non-blocking):** do we need this even though we provide `/statistics` through Open API ?

Reply:
yes, bc the next match uses this regex:
```
var clusterv1Regexp = regexp.MustCompile("/v1/cluster/*")
```
so the flow is that we first check the regex if we don't have a match then we fallback to OpenAPI, that's why I had to add this `r.URL.Path == "/v1/cluster/statistics"` check so that we would not fall into `cluster handler` but would fall into OpenAPI

Reply:
aah, right because it contains `/v1/cluster` gotcha